feat(Footer) : Footer page

Vyshnav001 opened the pull request.

Vyshnav001 committed:
feat(loading page)

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Add red color for AR instead of T

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Add Footer component to RootLayout in layout.tsx

Vyshnav001 committed:
feat(Footer) : Footer page

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

anishshobithps commented:
> @Vyshnav001 is attempting to deploy a commit to the **gdscsjec-devs** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=gdscsjec-devs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%2214fe5b850886c55aa8174ee261e57895782c53d9%22%7D%2C%22id%22%3A%22QmY4QgQ2eoAet38KuuBpDRWdjJfHFR7swoksdaJ9dSmbaM%22%2C%22org%22%3A%22tiarasjec%22%2C%22prId%22%3A20%2C%22repo%22%3A%22website%22%7D).

Not sure why is it asking me to authorize your PR everytime ?

anishshobithps commented:
![image](https://github.com/tiarasjec/website/assets/38991749/ed0aafec-098c-4a0e-a060-f18d2bb67765)

Make AR as red, and on Hover make Google DSC as Red only use the tiara_red, then add proper spacing between others.

Vyshnav001 commented:
For some reason Footer is not properly displaying in Event page

anishshobithps commented:
> For some reason Footer is not properly displaying in Event page

We will see to it, once #15 is merged!

Vyshnav001 committed:
Refactor Footer page

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'main' of https://github.com/Vyshnav001/frontend

Vyshnav001 commented:
@anishshobithps  should i keep underline color also as red ?or white is fine ?

Vyshnav001 committed:
Refactor Home component to add loading page till it fetches the image form api

anishshobithps commented:
> @anishshobithps should i keep underline color also as red ?or white is fine ?

Not sure if underline is going to look good ?

rabilrbl commented:
> > @Vyshnav001 is attempting to deploy a commit to the **gdscsjec-devs** Team on [Vercel](https://vercel.com).
> > A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=gdscsjec-devs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%2214fe5b850886c55aa8174ee261e57895782c53d9%22%7D%2C%22id%22%3A%22QmY4QgQ2eoAet38KuuBpDRWdjJfHFR7swoksdaJ9dSmbaM%22%2C%22org%22%3A%22tiarasjec%22%2C%22prId%22%3A20%2C%22repo%22%3A%22website%22%7D).
> 
> Not sure why is it asking me to authorize your PR everytime ?

He's using personal forked branch

anishshobithps commented on src/components/widgets/Footer.tsx:
```diff
@@ -0,0 +1,40 @@
+            <div className="container w-full px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+                <a href="/" className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+                    <Image src={SJECLogo} alt="SJEC Logo" className="w-10 h-15" width={70}/>
+                    <span className=" font-tiara ml-3 text-xl text-white ">Ti</span><span className=" font-tiara text-xl text-tiara_red ">ar</span><span className=" font-tiara text-xl text-white ">a 2024</span>
```
the font is quite tightly packed, use `leading-wider` iirc, that time text will be a bit wide and look better, Also I think we can remove SJEC logo at the footer ?  cc @rabilrbl 

Also, @rabilrbl will provide you with Privacy policy etc... make a separate route for it, as in like /privacy, /refund, /terms

You can use `prose` from `tailwindcss/typography` to format neatly.

rabilrbl replied:
@anishshobithps 
SJEC logo in footer not required

I only have T&C, come up with own privacy policy

anishshobithps reviewed (changes requested):
Also, the contact number should be changed, and the email is `tiara@sjec.ac.in`,

rabilrbl commented on src/components/widgets/Footer.tsx:
Can you please store all SVG icons under a folder in components?

Vyshnav001 committed:
Add EmailIcon component to src/components/icons/EmailIcon.tsx

Vyshnav001 committed:
Add InstagramIcon component to src/components/icons/InstagramIcon.tsx

Vyshnav001 committed:
Add PhoneIcon component to src/components/icons/PhoneIcon.tsx

Vyshnav001 committed:
Add WebsiteIcon component to src/components/icons/WebsiteIcon.tsx

Vyshnav001 committed:
update in Footer page

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 commented:
1. phone number removed 
2. email id changed
3. removed SJEC logo from the footer
4. added link to privacy,refund,terms
5. made folder to store all the SVG icons

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main

Vyshnav001 committed:
Merge branch 'tiarasjec:main' into main
